src/harness/reference_models/geo/zones.py

```diff
@@ -42,6 +42,7 @@
 # The reference files for extra zones.
 USBORDER_FILE = 'usborder.kmz'
 URBAN_AREAS_FILE = 'Urban_Areas_3601.kmz'
+USCANADA_BORDER_FILE = 'uscabdry_fine.kmz'
```

Comment:
Can we make this filename more descriptive and straightforward please ?

Reply:
Changed to "uscabdry_sampled.kmz"